main/options.c

```diff
@@ -2194,11 +2194,11 @@ static void saveIgnoreToken(const char * ignoreToken)
 	if(tokenEnd <= tokenBegin)
 		return;
 
-	
 	ignoredTokenInfo * info = (ignoredTokenInfo *)eMalloc(sizeof(ignoredTokenInfo));
 	
 	info->ignoreFollowingParenthesis = ignoreFollowingParenthesis;
 	info->replacement = replacement ? eStrdup(replacement) : NULL;
+	info->replacementLength = replacement ? strlen(replacement) : 0;
```

Comment:
Another approach is holding `replacement` in a vString.
vString maintains the length of string.